### PR TITLE
realease gui v1

### PR DESCRIPTION
**Pull request by wangxiaodong1**
[realease gui v1](https://github.com/cubefs/cubefs/pull/2315/commits/c026abb4cf7e2529244d363a87e51668e8c96465)

### CODE REVIEW

**Review by xiaochunhe — CHANGES_REQUESTED** [review]
add a README.md to introduce the basic information and installation process

File: frontend/vue.config.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+const { defineConfig } = require('@vue/cli-service')
+
+module.exports = defineConfig({
+  publicPath: '/portal',
+  transpileDependencies: true,
+  lintOnSave: false,
+  devServer: {
+    proxy: {
+      '/api': {
+        target: 'http://10.176.204.37:6007',
```
Comment (weilephp) [inline frontend/vue.config.js:11]: ip去掉

**Review by xiangcai1215 — APPROVED** [review]
LGTM

**Comment by hooklee2000** [comment]
In config.yml  the password of mysql  supports  plain or AES encrypted string.  
Maybe cfs-gui need provide an  -enc parameter to encrypt plain string and print encrypted string?

**Pull request merged**